src/phoenix/db/migrations/future_versions/cd164e83824f_users_and_tokens.py

```diff
@@ -6,137 +6,11 @@
-    # See # See https://docs.sqlalchemy.org/en/20/core/custom_types.html
-    cache_ok = True
-    impl = JSON_
-
-    def process_bind_param(self, value: Optional[Dict[str, Any]], _: Dialect) -> Dict[str, Any]:
-        return value if isinstance(value, dict) else {}
-
-
-class JsonList(TypeDecorator[List[Any]]):
-    # See # See https://docs.sqlalchemy.org/en/20/core/custom_types.html
-    cache_ok = True
-    impl = JSON_
-
-    def process_bind_param(self, value: Optional[List[Any]], _: Dialect) -> List[Any]:
-        return value if isinstance(value, list) else []
-
-
-class UtcTimeStamp(TypeDecorator[datetime]):
-    # See # See https://docs.sqlalchemy.org/en/20/core/custom_types.html
-    cache_ok = True
-    impl = TIMESTAMP(timezone=True)
-
-    def process_bind_param(self, value: Optional[datetime], _: Dialect) -> Optional[datetime]:
-        return normalize_datetime(value)
-
-    def process_result_value(self, value: Optional[Any], _: Dialect) -> Optional[datetime]:
-        return normalize_datetime(value, timezone.utc)
-
-
-class ExperimentRunOutput(TypedDict, total=False):
-    task_output: Any
-
-
-class Base(DeclarativeBase):
-    # Enforce best practices for naming constraints
-    # https://alembic.sqlalchemy.org/en/latest/naming.html#integration-of-naming-conventions-into-operations-autogenerate
-    metadata = MetaData(
-        naming_convention={
-            "ix": "ix_%(table_name)s_%(column_0_N_name)s",
-            "uq": "uq_%(table_name)s_%(column_0_N_name)s",
-            "ck": "ck_%(table_name)s_`%(constraint_name)s`",
-            "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
-            "pk": "pk_%(table_name)s",
-        }
-    )
-    type_annotation_map = {
-        Dict[str, Any]: JsonDict,
-        List[Dict[str, Any]]: JsonList,
-        ExperimentRunOutput: JsonDict,
-    }
-
-
-class UserRole(Base):
-    __tablename__ = "user_roles"
-    id: Mapped[int] = mapped_column(primary_key=True)
-    name: Mapped[str] = mapped_column(unique=True)
-
-
-class User(Base):
-    __tablename__ = "users"
-    id: Mapped[int] = mapped_column(primary_key=True)
-    user_role_id: Mapped[int] = mapped_column(
-        ForeignKey("user_roles.id"),
-        index=True,
-    )
-    username: Mapped[Optional[str]] = mapped_column(nullable=True, unique=True, index=True)
-    email: Mapped[str] = mapped_column(nullable=False, unique=True, index=True)
-    auth_method: Mapped[str] = mapped_column(
-        CheckConstraint("auth_method IN ('LOCAL')", name="valid_auth_method")
-    )
-    password_hash: Mapped[Optional[str]]
-    reset_password: Mapped[bool]
-    created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
-    updated_at: Mapped[datetime] = mapped_column(
-        UtcTimeStamp, server_default=func.now(), onupdate=func.now()
-    )
-    deleted_at: Mapped[Optional[datetime]] = mapped_column(UtcTimeStamp)
-
+from sqlalchemy import ScalarSelect, Table, insert
```

Comment:
Not trying to be overly nitpicky but prefer changes to files like these to be split out into separate PRs if possible. It makes PRs easier to review and easier to roll back.

Reply:
This main change here (line 171) is actually necessary to make the integration test work. The code that's deleted, even though it looks like a lot, is only the consequence of two lines of change (on line 66 and 76).

Reply:
sg